### ISSUE TITLE
The CI is always pending

### ISSUE THREAD

**Issue by testforstephen** [eclipse/lsp4j#390]
The CI server http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/ looks like broken. The last success build is Oct. 14. And the new build jobs are all pending. Is there anyone to fix it?

**Comment by testforstephen** [comment]
@jonahgraham I just found the site https://download.eclipse.org/lsp4j/builds/, is `master-new` folder built from the latest master branch code?

**Comment by jonahgraham** [comment]
@testforstephen Thank you for bringing this to my attention.  https://download.eclipse.org/lsp4j/builds/master/ is the correct URL for the latest master branch build and it now has the fixes that have been merged to master.

There was a script error and I have fixed that in 4da62c7306de6384ce6a528fc33654ca955bce2b. (the master-new should be transient and moved to master as part of the build). The moving the CI builds to https://ci.eclipse.org/lsp4j/job/lsp4j-multi-build/job/master/ is part of #369 - but it isn't quite finished.
